server/common/src/main/java/org/eclipse/lsp/cobol/common/mapping/ExtendedDocument.java

```diff
@@ -54,6 +55,29 @@ public String getUri() {
     return baseText.getUri();
   }
 
+  /**
+   * Returns original text, situated between the start and the end lines from the locality range
+   * @param locality - the text locality
+   * @return string value that represents a text from the original document
+   */
+  public String getBaseText(Locality locality) {
```

Comment:
Do you think it will make sense to have a unit test for the method?

Reply:
Yep, it make sense